src/v/features/feature_table.h

```diff
@@ -244,259 +252,259 @@ static_assert(is_major_version_upgrade(
 
 constexpr static std::array feature_schema{
   feature_spec{
-    cluster::cluster_version{5},
+    to_cluster_version(release_version::v22_2_1),
     "serde_raft_0",
     feature::serde_raft_0,
     feature_spec::available_policy::always,
     feature_spec::prepare_policy::always},
   feature_spec{
-    cluster::cluster_version{5},
+    to_cluster_version(release_version::v22_2_1),
```

Comment:
arguably the conversion be pushed down into the constructor I think

Reply:
Yeah that's much nicer 🔥